### DISCUSSION TITLE
Nice one - thanks for your hard work on this :)

### DISCUSSION THREAD

**Discussion by simontimothy** [AdnanHodzic/auto-cpufreq#232] [General]
Nice one - thanks for your hard work on this :)

**Comment by AdnanHodzic** [comment]
Most welcome, consider [donating ](https://github.com/AdnanHodzic/auto-cpufreq#donate)and supporting the project :)

**Comment by cnske** [comment]
I can only agree. Just wonder if there is the plan to create a PPA or similar for that, to make updating more easy in the future.
